src/js/renderers/Primitive/AlignmentMatrixControl.js

```diff
@@ -0,0 +1,96 @@
+import React from "react";
+import { withJsonFormsControlProps } from "@jsonforms/react";
+import { 
+	rankWith, 
+	isStringControl,
+	and,
+	optionIs 
+} from "@jsonforms/core";
+import { 
+	Button,
+	Popover,
+	Tooltip,	
+    FlexItem,
+	Dropdown,
+	SlotFillProvider, 
+	__experimentalHStack as HStack,
+	__experimentalSpacer as Spacer,
+	__experimentalDropdownContentWrapper as DropdownContentWrapper,
+	__experimentalAlignmentMatrixControl as AlignmentMatrixControl,
+} from '@wordpress/components';
+
+const AlignmentMatrixControlRenderer = ( props ) => {
+	const {
+		id,
+		description,
+		label,
+		path,
+		data,
+		handleChange
+	} = props;
+  
+  	return ( 
+		<>
+			<Spacer paddingY={ 2 } marginBottom={ 0 }>
+				<HStack justify="space-between">
+					<FlexItem>
+					{ description ? (
+						<Tooltip text={ description }>
+						<label htmlFor={ id }>
+							{ label }
+						</label>
+					</Tooltip>
+					) : ( 
+						<label htmlFor={ id }>
+							{ label }
+						</label> 
+					) }
+					</FlexItem>
+					<FlexItem>
+						<Dropdown
+							popoverProps={ {
+								placement: 'left-start',
+								offset: 36,
+								shift: true,
+							} }
+							className="my-container-class-name"
+							contentClassName="my-dropdown-content-classname"
+							renderToggle={ ( { isOpen, onToggle } ) => (
+								<Button
+									onClick={ onToggle }
+									aria-expanded={ isOpen }
+									icon={ 
+										<AlignmentMatrixControl.Icon
```

Comment:
The AligmentMatrixControl Icon can change its content based on the value to show different state of the alignment